packages/dds/shared-object-base/src/remoteObjectHandle.ts

```diff
@@ -20,10 +20,6 @@ import { FluidHandleBase, responseToException } from "@fluidframework/runtime-ut
  * IFluidHandle can be retrieved by calling `get` on it.
  */
 export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
-	public get IFluidHandleContext() {
```

Comment:
Unneeded override?

Reply:
The base (abstract) class doesn't have it, actually. It was just not referenced and the class is not public API so I just removed it.